### DISCUSSION TITLE
Getting 'This Serverless Function has crashed' in Vercel

### DISCUSSION THREAD

**Discussion by RageshAntony** [vercel/community#6565] [Help] [answered]
### Summary

I deployed my forked repo in Vercel but I got this error

<img width="518" alt="image" src="https://github.com/cool-dev-guy/vidsrc-api/assets/36100215/55c8c18c-58d9-4c6e-9e61-afb1f18e204d">

Logs:

<img width="1418" alt="image" src="https://github.com/cool-dev-guy/vidsrc-api/assets/36100215/c8b9dd5c-7fbb-406a-ab4f-3cf2f27221da">

Why? I followed your steps. (I didn't get any build error)

This is my repo:
https://github.com/RageshAntony/vidsrc-api-stermio.git

I am using FastAPI and Redis

please help me

### Steps to Reproduce

https://github.com/RageshAntony/vidsrc-api-stermio.git

Deploy this repo in Vercel

**Comment by versecafe** [comment, marked as answer]
@RageshAntony 500 means runtime error on the server so if you have any actual issues in your code such as trying to read non-existant values this can happen, but it should be possible to replicate in local development to get proper error logs, you can also try wrapping areas of the code in try catch blocks to find the issue, If you can get some more specific errors beyond the general LAMDA_WARNING that would help a ton.

**Reply by RageshAntony** [reply]
@versecafe 

I got the issue. it's Redis connection error. fixed it.
